src/internal/contextFile.go

```diff
@@ -90,9 +90,16 @@ func parseContextFile(dir string, data []byte) (Contexts, error) {
 	}
 
 	if dir != "" {
-		contexts.Context = resolveContextPaths(dir, contexts.Context)
+		var err error
```

Comment:
this is becoming a little noisy. Maybe a helper function `func resolvePaths(c Contexts) (Contexts, error)` would help?